app/views/pages/_placeholder.html.erb

```diff
@@ -0,0 +1,13 @@
+<% title "#{title} #{community_qualified_name}" %>
+
+<div class="blank-space"></div>
+<div class="container article">
+  <div class="title">
+    <h1><%= title %></h1>
+  </div>
+  <div class="body">
+    <p>
+      The production <%= title %> page is a page generated at <a href="/internal/pages">/internal/pages</a>! This one is just a placeholder.
```

Comment:
Should we alter this to ```The production <%= title %> page should be a page generated at <a href="/internal/pages">/internal/pages</a>! This one is just a placeholder.```? or something similar.

The current message works for DEV but  if I was spinning up a new community I may think its already generated for me on internal/pages.

Reply:
Makes sense but since the tests are super slow and this is just a text change, maybe we can attach it to another PR :D

Reply:
That sounds good to me!